Userland/Applications/PixelPaint/FilterTool.cpp

```diff
@@ -0,0 +1,119 @@
+/*
+ * Copyright (c) 2021, Tobias Christiansen <tobyase@serenityos.org>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#include "FilterTool.h"
+#include "FilterParams.h"
+#include "ImageEditor.h"
+#include "Layer.h"
+#include <AK/String.h>
+#include <LibGUI/BoxLayout.h>
+#include <LibGUI/Button.h>
+#include <LibGUI/ComboBox.h>
+#include <LibGUI/ItemListModel.h>
+#include <LibGUI/Widget.h>
+#include <LibGfx/Filters/BoxBlurFilter.h>
+
+namespace PixelPaint {
+FilterTool::FilterTool()
+{
+}
```

Comment:
Consider marking the constructor `= default` since it doesn't do anything anyways.
I.e. changing `FilterTool();` -> `FilterTool() = default;` in the header and removing this constructor definition.